RELEASE/scripts/autoscend/auto_standard.ash

```diff
@@ -1,11 +1,11 @@
 script "standard.ash"
```

Comment:
there was an sl_standard.ash and a standard.ash. One was basically a subset of the other with the difference of using `sl_my_path` rather than `my_path`, so I got rid of the other one. pretty sure it was an oversight.